scripts/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM squidfunk/mkdocs-material:8.5.9
+
+# mike version 2 is required in order to enable ENV vars for the container. By the time mike 2 is released we'd
+# use a specific dev version: https://github.com/jimporter/mike/issues/125
+RUN cd "$(python -c 'import site; print(site.getsitepackages()[0])')" && \
```

Comment:
We need to CD into Python's site-packages directory because for some reason pip install ignores the --target or --root flags and installs the packages into the `./src` folder.

And our current directory is the mounted one. In the case of installing into the current directory, when the container gets build, mike works fine. But when we need to mount the folder once again for another `docker run`, it fails

Reply:
Thanks a lot!

Maybe, you're right that `python -m` is the key here. Not sure what's the difference :D